src/multi_stage/endogenous_retirement.jl

```diff
@@ -22,6 +22,51 @@ function get_retirement_stage(cur_stage::Int, lifetime::Int, stage_lens::Array{I
     return Int(ret_stage)
 end
 
+function update_cumulative_min_ret!(inputs_d::Dict,t::Int,Resource_Set::String,dfGen_Name::String,RetCap::Symbol)
+
+	CumRetCap = Symbol("Cum_"*String(RetCap));
+
+	if !isempty(inputs_d[1][Resource_Set])
+		if t==1
+			inputs_d[t][dfGen_Name][!,CumRetCap] = inputs_d[t][dfGen_Name][!,RetCap];
+		else
+			inputs_d[t][dfGen_Name][!,CumRetCap] = inputs_d[t-1][dfGen_Name][!,CumRetCap] + inputs_d[t][dfGen_Name][!,RetCap];
+		end
+	end
+
+end
+
+
+function compute_cumulative_min_retirements!(inputs_d::Dict,t::Int)
+
+	if isempty(inputs_d[1]["VRE_STOR"])
+		mytab =[("G","dfGen",:Min_Retired_Cap_MW),
+				("STOR_ALL","dfGen",:Min_Retired_Energy_Cap_MW),
+				("STOR_ASYMMETRIC","dfGen",:Min_Retired_Charge_Cap_MW)
+			]
+	else
+		mytab =[("G","dfGen",:Min_Retired_Cap_MW),
+				("STOR_ALL","dfGen",:Min_Retired_Energy_Cap_MW),
+				("STOR_ASYMMETRIC","dfGen",:Min_Retired_Charge_Cap_MW),
+				("VS_DC","dfVRE_STOR",:Min_Retired_Cap_Inverter_MW),
+				("VS_SOLAR","dfVRE_STOR",:Min_Retired_Cap_Solar_MW),
+				("VS_WIND","dfVRE_STOR",:Min_Retired_Cap_Wind_MW),
+				("VS_STOR","dfGen",:Min_Retired_Energy_Cap_MW),
+				("VS_ASYM_DC_DISCHARGE","dfVRE_STOR",:Min_Retired_Cap_Discharge_DC_MW),
+				("VS_ASYM_DC_CHARGE","dfVRE_STOR",:Min_Retired_Cap_Charge_DC_MW),
+				("VS_ASYM_AC_DISCHARGE","dfVRE_STOR",:Min_Retired_Cap_Discharge_AC_MW),
+				("VS_ASYM_AC_CHARGE","dfVRE_STOR",:Min_Retired_Cap_Charge_AC_MW)
+			]
+	end
```

Comment:
Is the first `mytab` a subset of the second one? Could this be converted to 
```
mytab = ...

if VRESTORE
   add new entries to my_tab
end
```

Then they'd only need to be listed once.